README.md

```diff
@@ -194,6 +204,53 @@ $view = $this->factory->create(EnumType::class, null, array(
 ))->createView();
 ```
 
+#### Twig extension
+
+This package come with a `enum_label` filter, available on the `EnumExtension` Twig class.
+You have to require the `twig/twig` package to get it working.
+
+The filter will try to return the constant label corresponding to the given value.
+
+It will try to translate it if possible. To enable translation, require the `symfony/translation` component
+and pass a `Symfony\Component\Translation\TranslationInterface` instance on the `EnumExtension` constructor.
+
+If not translation available, you will have the default label with class prefixing.
```

Comment:
No. Consider this as a missing translation key, like `trans` does.

Returning the label without prefix would be confusing because the end use will add the key on the translation file and... this will not work.